styles.css

```diff
@@ -5,6 +5,8 @@ body {
   font-family: 'Inter', sans-serif;
 }
```

Comment:
- [ ] Consider reducing the body margin to zero, as it is affecting the site.

![chrome-capture-2022-8-23](https://user-images.githubusercontent.com/52143815/192005072-b041d930-64d7-4a3e-9c1f-c65d7582308f.gif)